Sources/Bot/Services/GitHubService.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+import ReactiveSwift
+import Result
+
+public final class GitHubService {
+    private let eventsObserver: Signal<Event, NoError>.Observer
+    public let events: Signal<Event, NoError>
+
+    public init() {
+        (events, eventsObserver) = Signal.pipe()
+    }
+
+    public func handleEvent(from request: RequestProtocol) -> Result<Void, EventHandlingError> {
```

Comment:
This can be changed since there's not really any output but I think `Result` overall provides a stronger model in terms of expression and extension, if tomorrow we add some output we could extended it without refactoring each call site.